wpilibcExamples/src/main/cpp/examples/ElevatorSimulation/cpp/Robot.cpp

```diff
@@ -114,12 +124,15 @@ class Robot : public frc::TimedRobot {
       // Here, we run PID control like normal, with a constant setpoint of 30in.
       double pidOutput = m_controller.Calculate(m_encoder.GetDistance(),
                                                 units::meter_t{30_in}.value());
-      m_motor.SetVoltage(units::volt_t{pidOutput});
+      double feedForwardOutput = feedforward.Calculate(1) // velocity
```

Comment:
feedForwardOutput needs to be a `units::volt_t`.

What is `feedforward.Calculate(1)` for? This will always calculate the feedforward with a velocity of 1. You need to pass the desired velocity or acceleration to the feedforward.

The argument to feedforward needs to use a velocity unit type.